backend/src/api/candidates.js

```diff
@@ -37,6 +38,22 @@ router.post(
   })
 )
 
+router.get(
+  '/initialize-git',
```

Comment:
lets move this functionality into parseSheet, since we’ll be running that to generate the Candidates in the beginning. Also include setting the status. This shouldn’t be part of public facing API.

Reply:
I made an endpoint cause contributions change so we don't have to reparse everything to get an updated github contribution count.